galcheat/data/LSST.yaml

```diff
@@ -59,23 +59,23 @@ references:
     link: "https://www.lsst.org/about/tel-site/optical_design"
     comment: ""
   obscuration:
-    link: "https://arxiv.org/abs/0805.2366"
-    comment: "See https://github.com/aboucaud/galcheat/issues/38 for discussions"
+    link: "https://arxiv.org/abs/0805.2366v4.pdf"
```

Comment:
At several places you decided to change the link from the arXiv ref to the actual pdf version.
Is there a specific reason ?

On the one hand the paper version "fixes" the value it references, which is good.
One the other hand, when looking at https://arxiv.org/abs/0805.2366 one can see there exists a much recent version of the paper (v5) and thus I'm left wondering why you chose the v4 instead.

Reply:
For my case I found that most arXiv references at first were pdf links so I have put pdf links everywhere for consistency. Maybe @ismael-mendoza have thought the same ?

About the version it may be that we took it from WLD and that we did not double check it...

Reply:
Yes I agree that we should have abstract pages in case there is a newer version, but I also like "fixing" the value of the reference. How about the url to the abstract page, but in the comment we specify which version we took values from ?

Reply:
Sounds about right :)

Reply:
excellent I can push that change